src/bootstrap/install.rs

```diff
@@ -49,6 +49,12 @@ pub fn install(build: &Build, stage: u32, host: &str) {
         install_sh(&build, "docs", "rust-docs", stage, host, &prefix,
                    &docdir, &libdir, &mandir, &empty_dir);
     }
+
+    for target in build.config.target.iter().filter(|target| *target != host) {
```

Comment:
I added it because I'm not sure if `build.config.target` will always have the host target in it, if that is true, then sure, we can safely remove the filter.